couler/core/states.py

```diff
@@ -14,9 +14,14 @@
 from collections import OrderedDict
 
 from couler.core import utils
-from couler.core.proto_repr import cleanup_proto_workflow
 from couler.core.templates import Workflow
 
+try:
+    from couler.core.proto_repr import cleanup_proto_workflow
+except Exception:
+    # set cleanup_proto_workflow to an empty function for compatibility
+    cleanup_proto_workflow = lambda: None  # noqa: E731
```

Comment:
Are we handling `cleanup_proto_workflow = None` with meaningful error message somewhere?

Reply:
If importing fails, `cleanup_proto_workflow` will be set to an empty function, and following callings like `cleanup_proto_workflow()` will do nothing.